src/main/java/ch/njol/skript/Skript.java

```diff
@@ -923,6 +924,27 @@ public void run() {
 			}
 		}, this);
 
+		// Send a warning to console when the plugin is reloaded
+		Bukkit.getPluginManager().registerEvents(new Listener() {
+			@EventHandler
+			public void onServerReload(ServerLoadEvent event) {
+				if (!(event.getType() == ServerLoadEvent.LoadType.RELOAD)) return;
+
+				String warningMessage = "It appears that /reload or another plugin reloaded Skript. This is not supported behaviour and may cause issues.";
+				String restartMessage = "Please consider restarting the server instead.";
+
+				for (Player player : Bukkit.getOnlinePlayers()) {
+					if (!player.isOp()) return;
```

Comment:
return?? shouldn't this continue? (better to make it a positive condition anyway instead of a negative one)